libraries/standard/json/src/json.c

```diff
@@ -30,6 +29,9 @@ typedef enum
     true = 1, false = 0
 } bool_;
 
+#define isdigit_( x )    ( ( x >= '0' ) && ( x <= '9' ) )
+#define isspace_( x )    ( ( x == ' ' ) || ( x == '\t' ) || ( x == '\n' ) || ( x == '\r' ) )
```

Comment:
What about `\v` and `\f`?

Reply:
Not in the JSON spec.

Reply:
Might be worth calling this something like `json_isspace_` to note that its not the same as the standard `is_space` https://en.cppreference.com/w/cpp/string/byte/isspace

Reply:
I'll add a comment.